fastly/provider.go

```diff
@@ -24,17 +24,31 @@ func Provider() terraform.ResourceProvider {
 			},
 		},
 		DataSourcesMap: map[string]*schema.Resource{
-			"fastly_ip_ranges": dataSourceFastlyIPRanges(),
-			"fastly_waf_rules": dataSourceFastlyWAFRules(),
+			"fastly_ip_ranges":                    dataSourceFastlyIPRanges(),
+			"fastly_tls_activation":               dataSourceFastlyTLSActivation(),
+			"fastly_tls_activation_ids":           dataSourceFastlyTLSActivationIds(),
+			"fastly_tls_certificate":              dataSourceFastlyTLSCertificate(),
+			"fastly_tls_certificate_ids":          dataSourceFastlyTLSCertificateIDs(),
+			"fastly_tls_configuration":            dataSourceFastlyTLSConfiguration(),
+			"fastly_tls_configuration_ids":        dataSourceFastlyTLSConfigurationIDs(),
+			"fastly_tls_platform_certificate":     dataSourceFastlyTLSPlatformCertificate(),
+			"fastly_tls_platform_certificate_ids": dataSourceFastlyTLSPlatformCertificateIDs(),
+			"fastly_tls_private_key":              dataSourceFastlyTLSPrivateKey(),
+			"fastly_tls_private_key_ids":          dataSourceTLSPrivateKeyIDs(),
```

Comment:
Should this not be named consistently with the other data sources?
```suggestion
			"fastly_tls_private_key_ids":          dataSourceFastlyTLSPrivateKeyIDs(),
```

That said, I notice the 'resources' listed below aren't named consistently in that they don't mention the provider name `Fastly`. I'm not suggesting we change that as part of this PR as it's an unrelated refactoring, but more from an idiomatic terraform code perspective shouldn't they be `<resource|dataSource>Fastly<Name>`?

e.g.

```diff
- resourceServiceV1
+ resourceFastlyServiceV1
```

Reply:
Ahh yes good point, I did this check on filenames before opening the PR but missed the function names 😩